web/components/RoomConfigView.tsx

```diff
@@ -7,6 +7,9 @@ import { ConnectionCard } from "./ConnectionCard";
 import { GenericWebhookConfig } from "./roomConfig/GenericWebhookConfig";
 import { GitlabRepoConfig } from "./roomConfig/GitlabRepoConfig";
 
+import GitLabIcon from "../icons/gitlab.png";
```

Comment:
...what sorcery is this :o

Reply:
![tenor](https://user-images.githubusercontent.com/2072976/167100429-dfa2daf5-e64a-467a-9560-bb4093d02c5d.gif)